small corrections (#22)
linksplatform/Examples.Doublets.CRUD.DotNet: README.md

using Platform.Data.Doublets;
using Platform.Data.Doublets.Memory.United.Generic;

// A doublet links store is mapped to the "db.links" file:
using var links = new UnitedMemoryLinks<uint>("db.links");

// Creating a doublet link: 
var link = links.Create();

// The link is updated to reference itself twice (as a source and as a target):
link = links.Update(link, link, link); // The passed arguments are: an updated address, a new source, and a new target

// Read operations:
Console.WriteLine($"The number of links in the data store is {links.Count()}.");
Console.WriteLine("Data store contents:");
// The arguments of a query are restrictions: on address, on source, on target

> var any = links.Constants.Any; // "Links.Constants.Any" means that the query (in our example) doesn't recieve any link address restriction

Reviewer: ```suggestion
var any = links.Constants.Any; // Means that there is no restriction on link address
```
